coderd/notifications/manager.go

```diff
@@ -56,7 +59,13 @@ type Manager struct {
 //
 // helpers is a map of template helpers which are used to customize notification messages to use global settings like
 // access URL etc.
-func NewManager(cfg codersdk.NotificationsConfig, store Store, log slog.Logger) (*Manager, error) {
+func NewManager(cfg codersdk.NotificationsConfig, store Store, metrics *Metrics, log slog.Logger) (*Manager, error) {
+	// TODO(dannyk): add the ability to use multiple notification methods.
+	var method database.NotificationMethod
+	if err := method.Scan(cfg.Method.String()); err != nil {
+		return nil, xerrors.Errorf("given notification method %q is invalid", cfg.Method)
```

Comment:
Nit: simplify (given is given 😄?)
```suggestion
		return nil, xerrors.Errorf("notification method %q is invalid", cfg.Method)
```